### PR TITLE
Populate data set

### PR DESCRIPTION
**Pull request by cookel2**
### What

Jira ticket: https://jira.ons.gov.uk/browse/DIS-1899

Following the approach to the seeding of fake data, this ticket is to flesh out the data set of fake data for the upstream stub. This should include a variety of:
 * content types
 * releases vs non-releases
 ** release statuses
 ** release moves / dates
 * future and past release dates

### How to review

Run the service locally and check that the returned JSON covers the different types of data as required:

Run as follows:

```shell
make debug
```

Check the following URL, which should return a list of 36 resources containing the various data values:

http://localhost:29600/resources

### Who can review

Anyone but me.

### CODE REVIEW

File: data/json_files/release_future_date_change.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "1245"
+  ],
+  "language": "latest language",
+  "survey": "latest survey",
+  "canonical_topic": "latest canonical topic",
+  "cancelled": false,
+  "finalised": true,
+  "published": false,
+  "date_changes": ["a change_notice", "2059-07-01T12:07:20Z"],
+  "provisional_date": "0001-03-01T00:00:00Z"
```
Comment (lindenmckenzie) [inline data/json_files/release_future_date_change.json:23]: `provisional_date` is actually more of a genuine string format, like "March 2025" or "Late September 2026"

File: data/json_files/release_future_date_change.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "uri": "/employmentandlabourmarket/peoplenotinwork/unemployment/datasets/unemploymentbypreviousoccupationunem02",
+  "uri_old": "/an/old/uri",
```
Comment (lindenmckenzie) [inline data/json_files/release_future_date_change.json:3]: `uri_old` is going to do insert and delete so I would remove these from the datastore and they can be added as part of the renaming task.

File: data/json_files/release_future_date_change.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "content_type": "release",
+  "cdid": "ASELECTIONOFNUMBERSANDLETTERS123",
```
Comment (lindenmckenzie) [inline data/json_files/release_future_date_change.json:5]: I think somewhere else, possibly search-api we do validation on cdid and dataset_id:

https://github.com/ONSdigital/dp-search-api/blob/e6a10527f96c88c4bbe2a52e945e7a149127da3b/api/search.go#L138

https://github.com/ONSdigital/dp-search-api/blob/e6a10527f96c88c4bbe2a52e945e7a149127da3b/api/search.go#L415

So will want to match that - really we should do this in the pipeline as well but that's not the focus of this ticket.

**Review by lindenmckenzie — COMMENTED** [review]
Have left some broad comments on one but they probably need reflecting out to the rest.

File: data/json_files/release_future_date_change.json (hunk 1 above)
Comment (cookel2) [reply data/json_files/release_future_date_change.json]: 👍 done

File: data/json_files/release_future_date_change.json (hunk 2 above)
Comment (cookel2) [reply data/json_files/release_future_date_change.json:3]: BTW I notice that the release fields are not currently appearing in the JSON when it runs - that's the same in the main branch too.

File: data/json_files/release_future_date_change.json (hunk 1 above)
Comment (cookel2) [reply data/json_files/release_future_date_change.json]: BTW I notice that the release fields are not currently appearing in the JSON when it runs - that's the same in the main branch too.

File: data/json_files/release_future_date_change.json (hunk 2 above)
Comment (cookel2) [reply data/json_files/release_future_date_change.json:3]: Should I remove the name of the field too or just the value?

Comment (lindenmckenzie) [reply data/json_files/release_future_date_change.json:3]: https://jira.ons.gov.uk/browse/DIS-2064

Comment (lindenmckenzie) [reply data/json_files/release_future_date_change.json:3]: it's not down as `omitempty` so you can either just remove all the values or set it to `omitempty` and remove the key as well.

Comment (cookel2) [reply data/json_files/release_future_date_change.json]: 👍 - I've removed all the values and set it to omitempty

File: data/json_files/release_future_date_change.json (hunk 3 above)
Comment (cookel2) [reply data/json_files/release_future_date_change.json]: It looks like the dataset_id already meets the spec (it only has to be more than 1 character long). I've amended the CDID to be 5 characters long as required.

**Review by lindenmckenzie — APPROVED** [review]

**Pull request merged**